index.html

```diff
@@ -0,0 +1,571 @@
+
+### Backus-Naur Form for versions and version tags
+
+```plaintext
+#::[BNF grammar for SemVer specifiers]
+‹version tag› ::= "v" ‹version›
+
+‹version› ::= ‹major› "." ‹minor›
+
+‹major› ::= ‹numeric identifier›
+‹minor› ::= ‹numeric identifier›
+‹numeric identifier› ::= "0"
+                       | ‹positive digit›
+                       | ‹positive digit› ‹digits›
+‹digits› ::= ‹digit›
+           | ‹digit› ‹digits›
+
+‹digit› ::= "0"
+          | ‹positive digit›
+‹positive digit› ::= "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
+```
+
+## Extensibility
+
+Any directives, scalars, enums, input objects, and other schema elements defined by specs **may** be **extensible**. This means that their definitions within a core schema document do not have to exactly match definitions provided by specs. Instead, definitions should match all usages within the document.
+
+Standard extensibility rules:
+- **Directives** may be defined as taking additional arguments, provided those arguments are [prefixed](#prefixing) with the name of a document feature followed by two underscores ("`__`"). These arguments are interpreted as metadata-on-metadata, and are routed to the declaring spec for interpretation
+- **Enums** defined in features may include values of any name. For extensible enums, the space of **unprefixed** names belongs to the document. The space of **prefixed** names belongs to the spec with that prefix, and may be used to pass data between specs.
+- **Input Objects** may be defined with additional fields, provided those fields are [prefixed](#prefixing) with the name of a feature followed by two underscores.
+
+## Processing Schemas
+
+```mermaid
+#::[Example processing pipeline]
+graph LR
+  schema(["📄  Input Schema"]):::file-->proc("🤖 &nbsp;Processor")
+  proc-->output(["📄  Output Schema"]):::file
+  classDef file fill:none,color:#eee;
+  style proc fill:none,stroke:fuchsia,color:fuchsia;
+```
+
+A common use case is that of a processor which consumes a valid input schema and generates an output schema.
+
+The general guidance for processor behavior is: don't react to what you don't understand.
+
+Specifically, processors:
+  - **should** pass through `@core` directives which reference unknown feature URLs
+  - **should** pass through prefixed directives, types, and other schema elements  
+
+An exception to this is processors which prepare the schema for final public consumption. Such processors **may** choose to eliminate all unknown directives and prefixed types in order to hide schema implementation details within the published schema. This will impair the operation of tooling which relies on these directives—such tools will not be able to run on the output schema, so the benefits and costs of this kind of information hiding should be weighed carefully on a case-by-case basis.
+
+## Validations &amp; Algorithms
+
+### Bootstrapping *validates* Has Core Feature
```

Comment:
I'm not sure I understand the wording of this subsection title and the next one.  It reads as a non-grammatical series of words. I guess "Has Core Feature" is a thing which is the object of "validates" but it's hard to read.